actionpack/lib/action_dispatch/http/request.rb

```diff
@@ -85,6 +85,9 @@ def controller_class
       end
     end
 
+    # Returns true if the request has set a header matching the given key parameter
```

Comment:
@rafaelfranca I have made the requested change according to your review.
Thank you.